### PR TITLE
Readme for RFC repo

### PR DESCRIPTION
**Pull request by zmackie**
- Describes RFC process and links to text/0004-rfc-process.md

Signed-off-by: Zander Mackie <zmackie@gmail.com>

### CODE REVIEW

**Comment by hone** [comment]
Thanks for doing this!

File: README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# RFCs for the Cloud Native Buildpacks project
+
+Want to suggest a change to the Cloud Native Buildpacks project? Awesome!
+
+We follow an RFC (Request for Comments) process for substantial changes to the project.
+This process was originally described in the [RFC RFC](./text/0004-rfc-process.md) and is copied below:
+
+## RFC Process
+
+To get an RFC into Cloud Native Buildpacks, first the RFC needs to be merged into the RFC repo. Once an RFC is merged, it's considered 'active' and may be implemented to be included in the project. These steps will get an RFC to be considered:
```
Comment (jromero) [inline README.md]: Lots of RFCs. Would this make sense?

```suggestion
To get a proposal into Cloud Native Buildpacks, first, an RFC needs to be merged into the RFC repo. Once an RFC is merged, it's considered 'active' and may be implemented to be included in the project. These steps will get an RFC to be considered:
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# RFCs for the Cloud Native Buildpacks project
+
+Want to suggest a change to the Cloud Native Buildpacks project? Awesome!
+
+We follow an RFC (Request for Comments) process for substantial changes to the project.
+This process was originally described in the [RFC RFC](./text/0004-rfc-process.md) and is copied below:
```
Comment (jromero) [inline README.md]:
```suggestion
This process was originally described in the [RFC#4](./text/0004-rfc-process.md) and is copied below:
```

File: README.md (hunk 1 above)
Comment (zmackie) [reply README.md]: This language is copied directly from RFC#4, so unless we're amending that I think it should just be a direct copy.

Comment (hone) [reply README.md]: I think it's also ok to update english/wording without needing to touch the RFC.

**Comment by zmackie** [comment]
@jromero Wanna give this one more look to make sure your changes have been addressed?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# RFCs for the Cloud Native Buildpacks project
```
Comment (hone) [inline README.md:1]:
```suggestion
# Cloud Native Buildpacks RFCs - [Active RFCs](https://github.com/pulls?q=is%3Aopen+is%3Apr+archived%3Afalse+-label%3Ahold+repo%3Abuildpacks%2Frfcs+draft%3Afalse), [Finalizing RFCs](https://github.com/pulls?q=is%3Aopen+is%3Apr+archived%3Afalse+label%3A%22Final+Comment+Period%22+repo%3Abuildpacks%2Frfcs+draft%3Afalse+)
```

Comment (hone) [reply README.md:1]: Let's add links to Active RFCs and ones in FCP.

**Comment by zmackie** [comment]
@hone Done

**Pull request merged**